messaging/WFLY-6684_jms_bridge_statistics.adoc

```diff
@@ -0,0 +1,66 @@
+= JMS Bridge should display statistics about messages that have been processed.
+:author:            Emmanuel Hugonnet
+:email:             ehugonne@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+An Artemis JMS Bridge consumes messages from a source JMS destination, and send them to a target JMS destination.
+Typically either the source or the target destinations are on different servers. The bridge can also be used to bridge messages from other non Artemis
+JMS servers, as long as they are JMS 1.1 compliant.
+The goal of this RFE is to display statistics on the number of processed messages and how many were aborted/rolled back.
```

Comment:
Please also add a note here saying, that subject of this RFE is only displaying statistics from Artemis.